tests/testament/tester.nim

```diff
@@ -104,6 +105,12 @@ proc callCompiler(cmdTemplate, filename, options: string,
   elif suc =~ pegSuccess:
     result.err = reSuccess
 
+  if result.err == reNimcCrash and
+     ("Your platform is not supported" in result.msg or
+      "cannot open 'sdl'" in result.msg or
+      "cannot open 'opengl'" in result.msg):
```

Comment:
To the [sdl2](https://github.com/nim-lang/sdl2) and [opengl](https://github.com/nim-lang/opengl) repos I guess.